upgrade/upd_2.5.11-to-2.5.12/index.php

```
@@ -79,6 +79,53 @@ public function apply_deletephpmailer()
+        $result = $migrate->executeQueue(true);
+        if (false === $result) {
+            $this->logs[] = sprintf(
+                'Migration of users.actkey column failed. Error: %s - %s',
+                $migrate->getLastErrNo(),
+                $migrate->getLastError(),
+            );
+            return false;
+        }
+
+        return true;
+    }
```

PHPMD (2.15.0)

[error] The method apply_expandactkey is not named in camelCase. (undefined)

(CamelCaseMethodName)